contrib/pg_tde/documentation/docs/tde.md

```diff
@@ -18,11 +18,11 @@ The encryption process is the following:
 
 ![image](_images/tde-flow.png)
 
-When a user creates an encrypted table using `pg_tde`, a new random key is generated for that table using the AES128 (AES-ECB) cipher algorithm. This key is used to encrypt all data the user inserts in that table. Eventually the encrypted data gets stored in the underlying storage.
+When a user creates an encrypted table using `pg_tde`, a new random key is generated internally for that table using the AES128 (AES-ECB) cipher algorithm. This key is used to encrypt all data the user inserts in that table. Eventually the encrypted data gets stored in the underlying storage.
```

Comment:
AES-CBC

Reply:
Also, the key isn't generated using the algorithm, the key is just a securely random key. The encryption algorithm used with the key is `AES-CBC`